graphics/digital_differential_analyzer_line_drawing.py

```diff
@@ -0,0 +1,48 @@
+import matplotlib.pyplot as plt
+
+
+def dda_line(x1, y1, x2, y2):
```

Comment:
1. Like I said earlier, please do not abbreviate "digital differential analyzer".
2. Like the algorithm-keeper bot said, please add type hints and doctests.